e2e-polybft/e2e/nexus_bridge_test.go

```diff
@@ -1891,3 +1891,196 @@ func convertToEthValues(sendAmount uint64) (uint64, *big.Int) {
 
 	return sendAmountDfm.Uint64(), ethgo.Ether(sendAmount)
 }
+
+func TestE2E_ApexBridgeWithNexus_BatchFailed_Test1(t *testing.T) {
```

Comment:
I looks like these 3 tests are using almost identical code. Can we refactor this? Maybe use one function with different parameters or one function which receivers different functions for custom logic?